src/css/api_docs/README.md

```diff
@@ -0,0 +1,40 @@
+### What is in this folder?
+
+The CSS in this folder is applied to the content sections of API docs.
+It is generated from the pydata sphinx theme and vendored into Great Expectations.
+We have made some noted modifications to the CSS from the pydata sphinx theme.
+
+### pydata-sphinx-theme License
+
+Below is a copy of the pydata-sphinx-theme 
+license obtained from https://github.com/pydata/pydata-sphinx-theme/blob/main/LICENSE
+
+BSD 3-Clause License
```

Comment:
Is this an appropriate place / method for including the license for pydata-sphinx-theme since we are using the styling?